src/Common/tests/Tests/System/StringTests.cs

```diff
@@ -4360,35 +4785,552 @@ public static void ToLowerInvariant(string s, string expected)
             Assert.Equal(expected, destination.ToString());
         }
 
-        [Theory]
-        [InlineData("")]
-        [InlineData("hello")]
-        public static void ToString(string s)
+        [Fact]
+        public static void ZeroLengthToLower()
         {
-            Assert.Same(s, s.ToString());
-            Assert.Same(s, s.ToString(null));
-            Assert.Same(s, s.ToString(CultureInfo.CurrentCulture));
+            char[] expectedSource = { 'a', 'B', 'c' };
+            char[] a = { 'a', 'B', 'c' };
+            var expectedDestination = new char[1] { 'a' };
+            var expectedDestinationString = new char[0];
+            Span<char> destination = new char[1] { 'a' };
 
-            Assert.Equal(s, s.AsSpan().ToString());
-        }
+            string s1 = new string(a, 2, 0);            
+            Assert.Equal(expectedDestinationString, s1.ToLowerInvariant().ToArray());            
```

Comment:
nit: How about just compare to an empty string and maybe rename `s1` to `emptyString`?
```C#
Assert.Equal("", s1.ToLowerInvariant());
```

Reply:
we set to empty some line below
```cs
 s1 = string.Empty;     
```
i think that the idea in old test is to test `startIndex,count` feature result in a empty string, i thought to add string based one. I agree with you on names...empty is better!

Reply:
> i think that the idea in old test is to test `startIndex,count` feature result in a empty string, i thought to add string based one. I agree with you on names...`emptyString` is better!Set empty here seems a duplicates, what do you think?

I am not suggesting changing s1 to string.Empty. I saw that assginment later. I am suggesting removing `expectedDestinationString` and in the assert compare s1 to "".

Something like this:
```C#
string emptyString = new string(a, 2, 0);
Assert.Equal("", emptyString .ToLowerInvariant());
...
// span stuff
...

emptyString = string.Empty;
Assert.Equal("", emptyString .ToLowerInvariant());
```